doc/api/crypto.md

```diff
@@ -1291,7 +1307,9 @@ object, the following additional properties can be passed:
   * `crypto.constants.RSA_PKCS1_PSS_PADDING`
 
   Note that `RSA_PKCS1_PSS_PADDING` will use MGF1 with the same hash function
-  used to sign the message as specified in section 3.1 of [RFC 4055][].
+  used to sign the message as specified in section 3.1 of [RFC 4055][], unless
+  an MGF1 hash function has been specified as part of the key in compliance with
+  section 3.3 of [RFC 4055][].
```

Comment:
in compliance with 4055, but in contravention of https://tools.ietf.org/html/rfc5756

Reply:
Yeah... I'm not sure what to do about that. It's weird.

Reply:
_shrug_. out of our control